src/test/scala/nl.knaw.dans.easy.multideposit/BlackBoxSpec.scala

```diff
@@ -30,21 +31,12 @@ import scala.xml.{ Elem, Node, NodeSeq, XML }
 // http://www.scalatest.org/user_guide/sharing_tests
 class BlackBoxSpec extends UnitSpec with MockFactory with CustomMatchers {
 
-  private val formats = """
-                          |application/postscript
-                          |application/rtf
-                          |application/pdf
-                          |application/msword
-                          |text/plain
-                          |text/html
-                          |text/sgml
-                          |text/xml
-                          |image/jpeg
-                          |image/gif
-                          |image/tiff
-                          |video/quicktime
-                          |video/mpeg1
-                        """.stripMargin.split("\n").map(_.trim).toSet
+  private val formatsFile: Path = Paths.get("src/main/assembly/dist/cfg/acceptedMediaTypes.txt").toAbsolutePath
+  private val formats =
+    if (Files.exists(formatsFile)) managed(Source.fromFile(formatsFile.toFile)).acquireAndGet(_.getLines.map(_.trim).toSet)
+    else Set.empty[String]
```

Comment:
maybe we should error here, since this file is expected to be here at all times!
You can use `fail(...)` for that, it's a failing function from ScalaTest.